scripts/qa_crowbarsetup.sh

```diff
@@ -1005,6 +1020,9 @@ EOF
         else
             complain 18 "You requested a HA setup but for this combination ($cloudsource : $slesdist) no HA setup is available."
         fi
+        if iscloudver 6plus; then
```

Comment:
Shouldn't that be in an `elif` ? Otherwise you'll never reach this point on a SLE12 node with HA.